### ISSUE TITLE
[med] Safari + VoiceOver: Entered value on the text field is not announced.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8619]
## Action Performed:
1. Open iOS Expensify application.
2. Open settings and navigate to the workspace using Swipe Gesture.
3. Open workspace by Double Tap and go to help button.   
4. New content will appear, navigate to the request a setup call using Swipe Gesture and Double Tap it.
5. Enter some values on the text field and listen to the screen reader announcement.           

## Expected Result:
Information on the text field which is entered by the user should be announced by the screen reader.

## Actual Result:
Entered information on the Password is not announced.

**Other Occurrences:**
Same issue repro on PR #8743, #8689, #8757, #8741, #8868, #8802

## Workaround:
Yes
## Area issue was found in:
Workspace -> Get assistance

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users might be confused whether the entered information is added or not.

## Suggested resolution:
Make sure the screen reader announce the information of the text field value once the user gives the input.                                                                           Refer to:              
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

## Platform:

- iOS

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/163124258-862d2589-600b-4bea-a290-7159af8dd37b.mp4
